app/src/main/java/currencyConverter/repository/ICurrencyRateModelRepository.java

```diff
@@ -0,0 +1,12 @@
+package currencyConverter.repository;
+
+import currencyConverter.exception.CurrencyRateFetchingException;
+import currencyConverter.model.CurrencyRateModel;
+
+public interface ICurrencyRateModelRepository {
+
+    CurrencyRateModel create(CurrencyRateModel model) throws CurrencyRateFetchingException;
```

Comment:
Can you pls explain, why does it return CurrencyRateModel, if whis method just save it into repository?

Reply:
Just because it is a Java style CRUD repository. It also very useful if we want to make code cleaner (just look forward the code where this method is used)